package.json

```diff
@@ -371,5 +374,20 @@
         "winston": "^3.2.1",
         "winston-transport": "^4.3.0",
         "xml2js": "^0.4.19"
+    },
+    "prettier": {
+        "printWidth": 120,
```

Comment:
Does it make sense to have rules like print width in both `prettier` and `tslint`? Or should we remove the corresponding rules from `tslint`?

Reply:
We could remove them from tslint, but then if you did not have the git hook you could still push up badly formatted code that builds, so I like having both